setup.cfg

```diff
@@ -1,9 +1,9 @@
 [metadata]
 name = rubrix
-# description = ...
-# long_description = ...
-# keywords = ...
-# license = ...
+description = Open-source tool for tracking, exploring and evolving data for AI projects.
+long_description = Rubrix is a tool for tracking and iterating on data for artificial intelligence projects. Rubrix focuses on enabling novel, human in the loop workflows involving data scientists, subject matter experts and ML/data engineers.
+keywords = data-science natural-language-processing artificial-intelligence knowledged-graph developers-tools human-in-the-loop mlops 
```

Comment:
```suggestion
keywords = data-science natural-language-processing text-labeling data-annotation artificial-intelligence knowledged-graph developers-tools human-in-the-loop mlops 
```

Reply:
We can include that keyword in Github's description too, if you want.